### ISSUE TITLE
Add back support for offline servers, for bungeecord/waterfall.

### ISSUE THREAD

**Issue by AncientZarko** [yummypasta/MinecraftManhunt#27]
## Description
Describe your proposed feature!
Support for this was removed in 1.2.4, it should be added back. So that the plugin is useable on Bungeecord servers, or prehaps reading of the UUID from the bungeecord.

**Comment by yummypasta** [comment]
Hi @AncientZarko,
Thanks for using the plugin. Sorry for the unclear wording— the plugin has never worked in offline mode, not just version 1.2.4. I will try to figure out how to make it compatible with offline mode, but I've found it quite hard to test because the server somehow caches users in online mode, as seen in #13, so I can't promise a fix anytime soon.

**Comment by AncientZarko** [comment]
I see I hope you can fix it, as this plugin is unusable with my Waterfall server.

**Comment by yummypasta** [comment]
> Not much of a feature request, but.
The plugin does not work with offline-mode, and that is fine.
If the "Bungee" boolean is enabled in Spigot.yml, the plugin can correctly pull the data it needs from Bungee.
I reported this thinking it was a problem with the plugin when it really was a problem with my config.
TLDR: Plugin works with Bungee, just don't be a brainlet.

\- @AncientZarko, #29

**Comment by yummypasta** [comment]
Support for offline mode has been added in [version 1.3.3.4](https://github.com/yummypasta/MinecraftManhunt/releases/tag/v1.3.3.4)!
